app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -215,7 +215,9 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
         throw new InternalError(InternalErrorReason.ServerError)
       }
 
-      if (response.statusCode === 400) {
+      // 425 code is returned by blockfrost API and indicates a full mempool
+      // https://docs.blockfrost.io/#tag/Cardano-Transactions/paths/~1tx~1submit/post
+      if (response.statusCode === 400 || response.statusCode === 425) {
```

Comment:
Doesn't this result in showing the users the
```
[InternalErrorReason.TransactionRejectedByNetwork]: () =>
    'TransactionRejectedByNetwork: Submitting the transaction into Cardano network failed. We received this error and we will investigate the cause.',
```

Doesn't it make sense to create a different message for the 425 case - explaining the congestion, etc.?

Reply:
actually yes, sorry I somehow thought I handled it correctly but apparently forgot to get back to it after experimenting and fixing the other stuff :)